LICENSE.txt

```diff
@@ -1,4 +1,5 @@
 BSD 3-clause license
+Copyright (c) 2022, IBM
```

Comment:
I don't know if order of copyright matters. But we add IBM's copyrights after conda-forge.